src/malloc/expand_heap.c

```diff
@@ -67,7 +67,7 @@ void *__expand_heap(size_t *pn)
 	size_t min = (size_t)PAGE_SIZE << mmap_step/2;
 	if (n < min) n = min;
 	void *area = enclave_mmap(0, n, 0, PROT_READ|PROT_WRITE, 1);
-	if (area == MAP_FAILED) return 0;
```

Comment:
Can this be turned into an mmap call?  I think we are now not using `malloc` before LKL starts (doesn't need to be in this PR)